source/agora/test/Flash.d

```diff
@@ -1266,7 +1279,7 @@ unittest
     const charlie_diego_chan_id_2 = charlie_diego_chan_id_2_res.value;
     log.info("Charlie Diego channel ID: {}", charlie_diego_chan_id_2);
     network.listener.waitUntilChannelState(charlie_diego_chan_id_2,
-        ChannelState.SettingUp, WK.Keys.C.address);
+        ChannelState.SettingUp, Height(0), WK.Keys.C.address);
```

Comment:
What height is the Ledger at this point in time?

Reply:
The height is 3 at that point. Do you mean the `Height(0)` is kind of confusing? Actually, the height is not used in case of `Height(0)`. I could use `Height.init` instead, which needs some modification in other places.

Reply:
Yes, I could. But it needs to modify many other lines calling `onChannelNotify` so I will revert this line by changing the prototype of `waitUntilChannelState` if it's not so weird. Additionally, I will raise another PR if I need to change all the lines calling `onChannelNotify`.

Reply:
Yes I think an overload with `Height` included will be less confusing.

Reply:
Ah, I meant to change the order of the parameters, but I will consider the overload also.